Debian Policy 9.8 Keyboard configuration

The (Linux) console is configured to make <-- generate DEL, and Delete generate ESC [ 3 ~.


--- a/sys/dev/kbd/kbdtables.h
+++ b/sys/dev/kbd/kbdtables.h
@@ -56,7 +56,7 @@
 /*0b*/{{  '0',   ')',   NOP,   NOP,   '0',   ')',   NOP,   NOP, }, 0x33,0x00 },
 /*0c*/{{  '-',   '_',  0x1F,  0x1F,   '-',   '_',  0x1F,  0x1F, }, 0x00,0x00 },
 /*0d*/{{  '=',   '+',   NOP,   NOP,   '=',   '+',   NOP,   NOP, }, 0x33,0x00 },
-/*0e*/{{ 0x08,  0x08,  0x7F,  0x7F,  0x08,  0x08,  0x7F,  0x7F, }, 0x00,0x00 },
+/*0e*/{{ 0x7F,  0x7F,  0x7F,  0x7F,  0x08,  0x08,  0x7F,  0x7F, }, 0x00,0x00 },
 /*0f*/{{ 0x09,  BTAB,   NOP,   NOP,  0x09,  BTAB,   NOP,   NOP, }, 0x77,0x00 },
 /*10*/{{  'q',   'Q',  0x11,  0x11,   'q',   'Q',  0x11,  0x11, }, 0x00,0x01 },
 /*11*/{{  'w',   'W',  0x17,  0x17,   'w',   'W',  0x17,  0x17, }, 0x00,0x01 },
@@ -125,7 +125,7 @@
 /*50*/{{ F(58),  '2',   '2',   '2',   '2',   '2',   '2',   '2', }, 0x80,0x02 },
 /*51*/{{ F(59),  '3',   '3',   '3',   '3',   '3',   '3',   '3', }, 0x80,0x02 },
 /*52*/{{ F(60),  '0',   '0',   '0',   '0',   '0',   '0',   '0', }, 0x80,0x02 },
-/*53*/{{ 0x7F,   '.',   '.',   '.',   '.',   '.',   RBT,   RBT, }, 0x03,0x02 },
+/*53*/{{ F(61),  '.',   '.',   '.',   '.',   '.',   RBT,   RBT, }, 0x83,0x02 },
 /*54*/{{  NOP,   NOP,   NOP,   NOP,   NOP,   NOP,   NOP,   NOP, }, 0xFF,0x00 },
 /*55*/{{  NOP,   NOP,   NOP,   NOP,   NOP,   NOP,   NOP,   NOP, }, 0xFF,0x00 },
 /*56*/{{  NOP,   NOP,   NOP,   NOP,   NOP,   NOP,   NOP,   NOP, }, 0xFF,0x00 },
@@ -226,7 +226,7 @@
 /* 49-52 */	{"\033[H", 3}, {"\033[A", 3}, {"\033[I", 3}, {"-"     , 1},
 /* 53-56 */	{"\033[D", 3}, {"\033[E", 3}, {"\033[C", 3}, {"+"     , 1},
 /* 57-60 */	{"\033[F", 3}, {"\033[B", 3}, {"\033[G", 3}, {"\033[L", 3},
-/* 61-64 */	{"\177", 1},   {"\033[J", 3}, {"\033[~", 3}, {"\033[}", 3},
+/* 61-64 */	{"\033[3~",4}, {"\033[J", 3}, {"\033[~", 3}, {"\033[}", 3},
 /* 65-68 */	{"", 0}      , {"", 0}      , {"", 0}      , {"", 0}      ,
 /* 69-72 */	{"", 0}      , {"", 0}      , {"", 0}      , {"", 0}      ,
 /* 73-76 */	{"", 0}      , {"", 0}      , {"", 0}      , {"", 0}      ,
